route-snapper/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "route-snapper"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+abstutil = { git = "https://github.com/a-b-street/abstreet" }
+console_error_panic_hook = "0.1.6"
+geojson = "0.24.0"
+geom = { git = "https://github.com/a-b-street/abstreet" }
+getrandom = { version = "0.2.3", features = ["js"] }
```

Comment:
There's a bunch of junk dependencies added in for not great reasons. `map_model` pulls in too much and needs extra workarounds on web. I might trim this down, but probably just making this work off osm2streets is the better next step